_publications/2017_jpf_dr.md

```diff
@@ -0,0 +1,88 @@
+---
+layout: publication
+# The quotes make the : possible, otherwise you can do it without quotes
+title: "Consistency-Aware Scheduling for Weakly Consistent Programs"
+# YYYY_CONFSHORTNAME_PAPERSHORTNAME
+key: 2017_jpf_dr
+
+# conference | journal | poster | workshop
+type: workshop
+redirect_from: /publications/2017_jpf_dr
+
+# The shortname is used for auto-generated titles
+shortname: JPF 2017
+# add a 2:1 aspect ratio (e.g., width: 400px, height: 200px) to the folder /assets/images/papers/
+image: 2017_jpf_dr.png
+# add a 2:1 aspect ratio teaser figure (e.g., width: 1200px, height: 600px) to the folder /assets/images/papers/
+image_large: 2017_jpf_dr_teaser.png
+
+# Authors in the "database" can be used with just the key (lastname). Others can be written properly.
+authors:
+- dabaghchian
+- rakamaric
+
+journal-short: JPF
+year: 2017
+
+# bibentry: article | inproceedings | phdthesis | book
+bibentry: article
+bib:
+  journal = ACM SIGSOFT Software Engineering Notes
+  series:
+  volume:
+  publisher: ACM
+  editor:
+  pages: 
+  doi: 
+  address:
+  url: 
+  number:
+  month:
+  note:
+
+# Add things like "Best Paper Award at InfoVis 2099, selected out of 4000 submissions"
+award:
+
+# Use if this paper is linked to an internal project. This will link to the project site
+# project: upset
+
+# Use this if you have an external project website
+external-project: https://pages.lip6.fr/syncfree/index.html
+
+# The reference to the video entry
+video:
+# The reference to the preview viedo entry
+#preview-video:
+
+# the preprint
+pdf: 2017_jpf_dr.pdf
```

Comment:
Use pdf from here: `/uusoc/res/soarlab/www-soarlab/html/publications`
The one you uploaded is messed up.

Reply:
Done.